app/routes/authenticated/dashboard/sbom/app-scans.ts

```diff
@@ -35,8 +35,22 @@ export default class AuthenticatedDashboardSbomAppScansRoute extends Route {
       scan_query = '',
     } = params;
 
+    const sbomProject = await this.store.findRecord(
+      'sbom-project',
+      sbom_project_id
+    );
+
+    const sbomPrjId = sbomProject.project.get('id');
+    const project = await this.store.findRecord('project', sbomPrjId as string);
+
+    const projectLastFile = await this.store.findRecord(
+      'file',
+      project.get('lastFile').id
+    );
```

Comment:
can we not do ?

```js
const projectLastFile = await sbomProject.get('project').get('lastFile')
```

Reply:
The associated records have to be explicitly fetched. Using the above returns `undefined`.